.github/workflows/s3Deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy to S3
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
should this be published on a new release instead?

Reply:
Agreed

Reply:
Done